src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: all $transition-duration;
```

Comment:
you must specify the property you apply transition to

Reply:
Thanks!